style.css

```diff
@@ -1,5 +1,11 @@
+@font-face {
+  font-family: HanaMinA;
+  src: url(HanaMinA.woff);
+}
+
 .nrGrammardata {
   background: white;
+  font-family: serif, HanaMinA;
```

Comment:
This will _always_ pick `serif` because it will always work as per the CSS spec, so you'll have to swap these around. The generic family class keyword always has to come last.

Reply:
The idea was if the `serif` font has CJK glyphs than the reader probably likes the default font and don't really need the HanaMinA substitute. But if the default `serif` doesn't have CJK glyphs, which is I guess common for learners, they have their nice readable latin font and only CJK glyphs fall back to HanaMinA.

Alternative is to have a good looking latin font before HanaMinA and serif at the end.

Reply:
But I have no intention of honoring the reader's locally installed fonts: the reason we're subsetting is to ensure that everyone sees the same thing, with a font that covers exactly what is necessary =)